browser/about_flags.cc

```diff
@@ -486,6 +502,33 @@ constexpr char kPlaylistDescription[] = "Enables Playlist";
 #define PLAYLIST_FEATURE_ENTRIES
 #endif
 
+#if defined(TOOLKIT_VIEWS)
+uint16_t AllowForDevVersion(uint16_t os) {
+#if BUILDFLAG(IS_WIN)
+  if (!install_static::InstallDetails::IsSet()) {
```

Comment:
if this is happening in unit tests then you're missing setup that needs to happen for channel. I don't think this is the right way to fix it